Modules/Loadable/VesselEditor/Logic/vtkSlicerVesselEditorLogic.cxx

```diff
@@ -405,4 +405,36 @@ ::SplitTubeInSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode, int tubeID, d
       break;
     }
   }
-}
\ No newline at end of file
+}
+
+//---------------------------------------------------------------------------
+void vtkSlicerVesselEditorLogic
+::RenumberTubesInSpatialObject(vtkMRMLSpatialObjectsNode* spatialNode)
+{
+  if (!spatialNode)
+  {
+    return ;
+  }
+
+  TubeNetType* spatialObject = spatialNode->GetSpatialObject();
+  char childName[] = "Tube";
+  TubeNetType::ChildrenListType* tubeList =
+    spatialObject->GetChildren(spatialObject->GetMaximumDepth(), childName);
+  int numberOfTubes = tubeList->size();
+  TubeIdType parentTubeId = numberOfTubes + 1000;
```

Comment:
This line seems odd to me.   Why not start at 0 and go up from there?   Starting at 1000+numberOfChildren might still cause number clashes, and your code should not be effected by number clashes.